packages/mux-player/src/template.ts

```diff
@@ -105,6 +103,12 @@ export const content = (props: MuxTemplateProps) => html`
         ? html`<track label="thumbnails" default kind="metadata" src="${props.storyboard}" />`
         : html``}
     </mux-video>
+    <media-poster-image
```

Comment:
question(non-blocking): From the looks of this refactor, is the expectation that (non-`<mux-player>`) users of our themes will now have to slot a component to be able to see a poster image? If yes, that feels like a code sniff, but I don't want to hold up merging this, so we can treat it as a followup.